web/modules/custom/dcamp_schedule/templates/schedule.html.twig

```diff
@@ -42,180 +42,197 @@
           <span class="lightning"></span>
           <span class="intermediate"></span>
         </td>
       </tr>
       <tr>
-        <td class="lightning english intermediate"><a href="/sessions/synchronize-your-drupal-data-carto-plopesc"
-                                                      target="_blank">Synchronize
-            your Drupal data with Carto <span class="author"> {{ 'by'|t }} Pablo López Escobés</span></a>
+        <td><a href="/sessions/integracion-continua-con-circleci-para-drupal-8-regiguren"
+                                                      target="_blank">Integración continua con CircleCI para Drupal 8
+            <span class="author"> {{ 'by'|t }} Rubén Egiguren</span></a>
           <span class="lightning"></span>
-          <span class="intermediate"></span>
-          <span class="english"></span>
+          <span class="basic"></span>
         </td>
-        <td class="lightning intermediate"><a
-              href="/sessions/lets-encrypt-o-como-tener-tu-pagina-https-tras-5-instrucciones-en-la-linea-de-comandos"
-              target="_blank">Let’s Encrypt o como tener tu página HTTPS tras 5 instrucciones en la línea de comandos by
-            <span>Luis Ortiz Ramos</span></a></td>
-        <td class="lightning basic"><a href="/sessions/why-drupal-davidbaltha" target="_blank">Why Drupal <span
-                class="author"> {{ 'by'|t }} David Hernández Ruiz</span></a>
+        <td><a
+              href="/sessions/drush-9-creemos-un-comando-en-10-minutos-capynet"
+              target="_blank">Drush 9 creemos un comando en 10 minutos
+            <span>Marcelo Tosco</span></a>
           <span class="lightning"></span>
-          <span class="basic"></span>
+          <span class="intermediate"></span>
         </td>
       </tr>
       <tr>
-        <th scope="row">16:00</th>
-        <td class="featured intermediate"><a
-              href="/sessions/seleccionando-un-cms-para-la-transformacion-por-que-drupal-javierespadas" target="_blank">Seleccionando
-            un CMS para la transformación: ¿Por qué Drupal? <span class="author"> {{ 'by'|t }}
-              Javier Espadas Bardón</span></a>
-          <span class="featured"></span>
-          <span class="intermediate"></span>
-        </td>
-        <td class="english intermediate"><a href="/sessions/test-driven-development-drupal-8-nuezweb" target="_blank">Test
-            Driven Development in Drupal 8 <span class="author"> {{ 'by'|t }} Teun Van Veggel</span></a>
-          <span class="english"></span>
-          <span class="intermediate"></span>
-        </td>
-        <td class="basic"><a href="/sessions/tu-drupal-esta-listo-lo-sabe-google-beagonpoz" target="_blank">Tu Drupal
-            está
-            listo… ¿Lo sabe Google? <span class="author"> {{ 'by'|t }} Beatriz González Pozo</span></a>
-          <span class="basic"></span>
-        </td>
+        <th scope="row">16:15</th>
+        <td  colspan="3">Descanso / Break</td>
```

Comment:
I rather put everything in English by default and let the translation system do the rest when someone views the site in Spanish. Otherwise we will have a weird mix of translated strings and strings in both languages with a forward slash.